be/src/olap/types.h

```diff
@@ -74,6 +74,7 @@ class TypeInfo {
     inline void set_to_min(void* buf) const { _set_to_min(buf); }
 
     inline uint32_t hash_code(const void* data, uint32_t seed) const { return _hash_code(data, seed); }
+    inline uint64_t hash_code64(const void* data, uint64_t seed) const { return _hash_code64(data, seed); }
```

Comment:
why do we need hash64?

Reply:
it used to calculate hash code of specific type data, used by bloom filter.